### PR TITLE
add documentation on services

### PR DESCRIPTION
**Pull request by senyosimpson**
### Summary of changes

Adds a section our support for Kubernetes Services

### CODE REVIEW

File: kubernetes/configuring-services.html.markerb [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+```
+kubectl apply -f service.yaml
+```
+
+## Fly.io connection Handlers
+
+Our platform supports a number of connection handlers. These are a sort of middleware that modify your connection before it reaches your application.
+More details on the available handlers can be found in our [documentation](https://fly.io/docs/networking/services/#connection-handlers). Connection
+handlers are supported via custom annotations to a Service object. The annotations have the form
```
Comment (andie787) [inline kubernetes/configuring-services.html.markerb:42]:
```suggestion
Fly.io connection handlers are processes that modify your connection before it reaches your application.
Learn more about [connection handlers](/docs/networking/services/#connection-handlers). Connection
handlers are supported with custom annotations to a Service object. The annotations have the form:
```

Comment (andie787) [reply kubernetes/configuring-services.html.markerb:42]: processes or middleware? not sure which word is best

File: kubernetes/configuring-services.html.markerb [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+```yaml
+"service.fly.io/<exposed port name/number>-handlers": "<handler1,handler2,...handlerN>"
+```
+
+The example below adds an HTTP and TLS handler to the port 443.
```
Comment (andie787) [inline kubernetes/configuring-services.html.markerb:48]:
```suggestion
The example below adds an HTTP and TLS handler for port 443.
```

File: kubernetes/configuring-services.html.markerb [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+## Not supported
+
+We currently do not support:
+
+* NodePort Services
+* UDP protocol
+
+## Related topics
+
+- [FKS Quickstart](/docs/kubernetes/fks-quickstart/)
```
Comment (andie787) [inline kubernetes/configuring-services.html.markerb:80]:
```suggestion
- [FKS Quickstart](/docs/kubernetes/fks-quickstart/)
- [Create an FKS cluster](/docs/kubernetes/clusters/)

```

**Review by andie787 — COMMENTED** [review]
Left some suggestions to take or leave!

File: kubernetes/configuring-services.html.markerb (hunk 1 above)
Comment (senyosimpson) [reply kubernetes/configuring-services.html.markerb:42]: hmm, i think we can actually just not give them a description. "Fly.io connections handlers modify your connection" reads well

**Pull request merged**